client/src/search-utils.ts

```diff
@@ -14,7 +14,7 @@ export function useFocusOnSlash(
     function focusOnSearchMaybe(event) {
       const input = inputRef.current;
       if (
-        event.code === "Slash" &&
+        event.key === "/" && !event.ctrlKey && !event.metaKey &&
```

Comment:
To satisfy prettier:
```suggestion
        event.key === "/" &&
        !event.ctrlKey &&
        !event.metaKey &&
```